src/config.ts

```diff
@@ -176,10 +183,18 @@ const buildAppContext = async (secrets: any): Promise<AppConfig> => {
           process.env.REVISIONS_REQUESTED_GUIDE_URL ||
           'https://docs.icgc-argo.org/docs/data-access/daco/approval#requested-revisions',
         dacoSurvey: process.env.DACO_SURVEY_URL || '#',
+        accessRenewalGuide:
+          process.env.ACCESS_RENEWAL_GUIDE ||
+          'https://docs.icgc-argo.org/docs/data-access/daco/renew-close#renewing-an-application',
       },
       reviewerFirstName: process.env.EMAIL_REVIEWER_FIRSTNAME || 'DACO',
       reviewerLastName: process.env.EMAIL_REVIEWER_LASTNAME || 'administrator',
     },
+    durations: {
```

Comment:
Making expiry durations configurable to aid in testing expiry features.

Reply:
:rocket: